src/styled/feature-card.ts

```diff
@@ -0,0 +1,129 @@
+import styled from '@emotion/styled';
+import theme from '@mdb/flora/theme';
+import { PillCategory } from '../types/pill-category';
+import { HorizontalRule, Pill, Tag, TypographyScale } from '@mdb/flora';
+import Image from 'next/image';
+import { pillColorMap } from './pill-color';
+
+const FeaturedCardWrapper = styled('div')`
+    padding: 32px;
+    height: max-content;
+    border: 1px solid #e7eeec;
```

Comment:
We should probably use the flora theme value for this color (`colors.black20 = '#e7eeec'`).